spec/classes/certs_foreman_proxy_content_spec.rb

```diff
@@ -15,4 +15,10 @@
   describe 'with default parameters' do
     it { should compile.with_all_deps }
   end
+
+  context 'with empty certs_tar' do
+    let(:params) { { certs_tar: '' , foreman_proxy_fqdn: 'example.com' } }
```

Comment:
We can probably leave out the fqdn parameter (and in the basic test too).

Reply:
Looks like basic tests needs the default fqdn for one scenario:
```
    Failure/Error: it { should compile.with_all_deps }
       error during compilation: Evaluation Error: Error while evaluating a Function Call, The hostname is the same as the provided hostname for the foreman-proxy at /home/travis/build/theforeman/puppet-certs/spec/fixtures/modules/certs/manifests/foreman_proxy_content.pp:22:5 on node travis-job-theforeman-puppet-certs-377315866.travisci.net
```
```
certs::qpid_client
  on centos-7-x86_64
    without parameters
    should compile into a catalogue without dependency cycles (FAILED - 1)
```

Reply:
Ah right :)